source/code/go/src/plugins/oms.go

```diff
@@ -958,6 +978,13 @@ func InitializePlugin(pluginConfPath string, agentVersion string) {
 		Log("ResourceID=%s", ResourceID)
 		Log("ResourceName=%s", ResourceName)
 	}
+	
+	containerRuntime = os.Getenv(ContainerRuntimeEnv)
+	Log("Container Runtime engine %s", containerRuntime)
+	isDockerContainerRuntimeEngine = false
+	if strings.EqualFold(containerRuntime, "docker") {
```

Comment:
make the comparison opposite (meaning check for containerd, if not default to docker), everywhere.

Reply:
since the default set as "docker" in main.sh so we are good on this.